packages/material-ui/src/InputLabel/InputLabel.d.ts

```diff
@@ -15,11 +15,14 @@ export interface InputLabelProps extends StandardProps<FormLabelProps, InputLabe
 
 export type InputLabelClassKey =
   | 'root'
+  | 'focused'
+  | 'disabled'
+  | 'error'
+  | 'required'
   | 'formControl'
   | 'marginDense'
   | 'shrink'
   | 'animated'
-  | 'contained'
```

Comment:
I'll admit that this was not caught by typescript. Just plain "look at left vs right file".